YouTube.sh

```diff
@@ -99,9 +101,9 @@ case $opt in
 		echo -n "> "
 		read END
 		echo ""
-		echo "Running script : youtube-dl -f best --yes-playlist --playlist-start $START --playlist-end $END -o \"/home/$user/Videos/YouTube/%(uploader)s/%(title)s-%(id)s.%(ext)s\" $URL"
+		echo "Running command: youtube-dl -f bestvideo[ext=mp4]+bestaudio --external-downloader aria2c --yes-playlist --playlist-start $START --playlist-end $END -o \"$HOME/Downloads/YouTube/%(uploader)s/%(upload_date)s_%(title)s.%(ext)s\" $URL"
```

Comment:
Edited to download bestvideo and bestaudio seperately with an external auto-remuxing downloader (aria2c).

Previous command of "best" just downloads the pre-muxed 720p version and will not support 1080p or 4K video.